play/fraction-bingo/compare/app.js

```diff
@@ -0,0 +1,253 @@
+    }
+    problem = fract1 + " ? " + fract2;
+    let n1 = 0, n2 = 0, d1 = 0, d2 = 0;
+    let arr1 = fract1.split("/");
+    n1 = parseInt(arr1[0]);
+    d1 = parseInt(arr1[1]);
+    let arr2 = fract2.split("/");
+    n2 = parseInt(arr2[0]);
+    d2 = parseInt(arr2[1]);
+    let f1 = n1/d1;
+    let f2 = n2/d2;
+    if (f1 > f2) {
+        correctOp = ">";
+    } else if (f1 < f2) {
+        correctOp = "<";
+    } else {
+        correctOp = "=";
+    }
+    document.getElementById("problem").innerHTML = problem;
+}
+
+function getRandomFraction(min, max) {
+    let num;
+    let numerator = Math.floor(Math.random() * (max - min + 1)) + min;
+    if (min == 0) {
+	min = 1;
+    }
+    let denominator = Math.floor(Math.random() * (max - min + 1)) + min;
+    //num = `${numerator}&frasl;${denominator}`;
+    num = numerator + "/" + denominator;
+    return num;
+}
+
+// Randomly fill board with <, >, and = 
+function fillBoard() {
+    table = document.getElementsByName(boardType)[0];
+    let operators = ["<", ">", "="];
+    let op = "";
+    for (var i = 0, row; row = table.rows[i]; i++) {
+	for (var j = 0, col; col = row.cells[j]; j++) {
+	    op = operators[Math.floor(Math.random() * 3)];                                 
+            col.className = "white";
+	    col.innerText = op;
+            col.onclick = function () {
+		checkAnswer(this);
+            };
+	}
+    }
+}
+
+function checkAnswer(cell) {
+    if (cell.className != "green") {
+        if (correctOp == cell.innerText) {
+	    score += 10;
+	    numSolved++;
+	    cell.className = "green";
+	    let b = checkBingo();
+	    if (!b) {
+		setTimeout(function() {
+			generateProblem();
+		}, 1000);
+	    }
+	}
+        else {
+	    if (score != 0) {
+		score -= 10;
+	    }
+	    $('.list-group').append("<li class='list-group-item'><h5 class='list-group-item-heading'>" + problem + "</h5> <p class='list-group-item-text'><b>Your Answer: </b>" + cell.innerText + "<br> <b\
+>Correct: </b>" + correctOp + "</p> </li>");
+       	    cell.className = "red";
+	    setTimeout(function() {
+	        cell.className = "white";
+	    }, 1000);
+        }
+	document.getElementById("score").innerHTML = "Score: " + score;
+    }
+}
+
+function solNotFound() {
+    noSolBtn = document.getElementById("noSolution");
+    for (var i = 0, row; row = table.rows[i]; i++) {
+        for (var j = 0, col; col = row.cells[j]; j++) {
+	    if (col.className == "white") { 
+	        if (correctOp == col.innerText) {
+		    if (score != 0) {
+			score -= 10;
+		    }
+		    document.getElementById("score").innerHTML = "Score: " + score;
+		    $('.list-group').append("<li class='list-group-item'><h5 class='list-group-item-heading'>" + problem + "</h5> <p class='list-group-item-text'><b>Your Answer: </b> Solution not on boar\
+d <br> <b>Correct: </b>" + correctOp + "</p> </li>");
+                    noSolBtn.style.backgroundColor = "red";
+                    setTimeout(function() {
```

Comment:
When doing setTimeout in one line a great idea is to use a fat arrow function like this:
```js
setTimeout(() => noSolBtn.style.backgroundColor = "", 1000);
```